### PR TITLE
[IDE] Remove "Begin completions" and "End completions" from test cases

### PR DESCRIPTION
**Pull request by ahoppen**
These test lines weren't actually providing any value and were annoying to write. Let's jut remove them

### CODE REVIEW

**Comment by ahoppen** [comment]
@swift-ci Please smoke test

File: test/IDE/complete_after_super.swift [hunk 1]
```diff
@@ -397,22 +362,19 @@ class Closures : SuperBaseA {
...
       // CLOSURE_2: Begin completions, 6 items
-      // CLOSURE_2: End completions
       super.#^CLOSURE_2?check=COMMON_BASE_A_DOT_CONTEXT;check=CLOSURE_2;check=NO_CONSTRUCTORS^#
     }
   }
 
   func baz() {
     let inner = { [weak self] in
       super.#^CLOSURE_CAPTURE_1?check=CLOSURE_CAPTURE_1;check=NO_SUPER_DECLS;check=NO_CONSTRUCTORS^#
-      // CLOSURE_CAPTURE_1-NOT: Begin completions
```
[the remark this answers is not in the document]
Comment (rintaro) [reply test/IDE/complete_after_super.swift:415]: Or just not to remove `-NOT: Begin completions` for now.

**Comment by ahoppen** [comment]
@swift-ci Please smoke test

**Comment by ahoppen** [comment]
@swift-ci Please smoke test

**Review by rintaro — APPROVED** [review]

**Pull request merged**